.circleci/config.yml

```diff
@@ -249,10 +249,11 @@ jobs:
     # version to keep this one working for now.
       - image: cimg/node:20.8.1-browsers
     <<: *desktop_defaults
+    resource_class: medium+
```

Comment:
Nice catch! And I see CI is 🟢  👏 

Also, from the docs:
>  It is best practice to specify a resource class as opposed to relying on a default.

Guessing the default is `medium` (4GB RAM), while `medium+` gives us 6GB? I wonder if there's any impact / downside (I guess cost, but perhaps it's not a big difference?) of just going straight to `large` / 8GB, given that also doesn't sound like a lot.

Reply:
>Guessing the default is medium (4GB RAM), while medium+ gives us 6GB?

Exactly.

>given that also doesn't sound like a lot.

Agreed. In comparison, the Windows builds have 15GB and macOS builds have 6GB. I haven't done the math to estimate how this could play out cost-wise (now that the desktop builds may see more activity), but figured we could at least start with `medium+` on Linux first. 👍